src/components/Footer.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <footer id="footer">Created by Jessica, Katie, and Nikki, with 🦑, 🐢, and 👖.</footer>
```

Comment:
Could add the BE-er names to this too!

Reply:
🦑

Reply:
dang I was not thinking this morning.  got them all in and their fave emojis.  Thank you so so much.